peps/pep-0784.rst

```diff
@@ -2,97 +2,112 @@ PEP: 784
+standard. This PEP proposes adding a new module to the Python standard library
+containing a Python wrapper around Meta's |zstd| library, the default
+implementation. Additionally, to avoid name collisions with packages on PyPI
+and to present a unified interface to Python users, compression modules in the
+standard library will be moved under a ``compression.*`` package.
+
+.. |zstd| replace:: ``zstd``
+.. _zstd: https://facebook.github.io/zstd/
+.. _Zstandard: https://facebook.github.io/zstd/
+
 
 Motivation
 ==========
 
-CPython has modules for several different compression formats, such as `zlib
-(DEFLATE) <https://docs.python.org/3/library/zlib.html>`_,
-`bzip2 <https://docs.python.org/3/library/bz2.html>`_,
-and `lzma <https://docs.python.org/3/library/lzma.html>`_, each widely used.
-Including popular compression algorithms matches Python's "batteries included"
-philosophy of incorporating widely useful standards and utilities. The last
-compression module added to the language was ``lzma``, added in Python 3.3.
+CPython has modules for several different compression formats, such as
+:mod:`zlib (DEFLATE) <zlib>`, :mod:`bzip2 <bz2>`, and :mod:`lzma <lzma>`,
+each widely used. Including popular compression algorithms matches Python's
+"batteries included" philosophy of incorporating widely useful standards and
+utilities. :mod:`!lzma` is the most recent such module, added in Python 3.3.
 
-Since then, Zstandard has become the modern de facto preferred compression
+Since then, Zstandard has become the modern *de facto* preferred compression
 library for both high performance compression and decompression attaining high
 compression ratios at reasonable CPU and memory cost. Zstandard achieves a much
 higher compression ratio than bzip2 or zlib (DEFLATE) while decompressing
 significantly faster than LZMA.
 
-Zstandard has seen `widespread adoption in many different areas of computing
-<https://facebook.github.io/zstd/#references>`_. The numerous hardware
-implementations demonstrate long-term commitment to Zstandard and an
-expectation that Zstandard will stay the de facto choice for compression for
-years to come. Zstandard compression is also implemented in both the ZFS and
-Btrfs filesystems.
+Zstandard has seen `widespread adoption`_ in many different areas of computing.
+The numerous hardware implementations demonstrate long-term commitment to
+Zstandard and an expectation that Zstandard will stay the *de facto* choice for
+compression for years to come. Zstandard compression is also implemented in
+both the ZFS_ and Btrfs_ filesystems.
 
 Zstandard's highly efficient compression has supplanted other modern
-compression formats, such as brotli, lzo, and ucl due to its highly efficient
-compression. While `LZ4 <https://lz4.org/>`_ is still used in very high
-throughput scenarios, Zstandard can also be used in some of these contexts.
+compression formats, such as brotli_, lzo_, and ucl_ due to its highly
```

Comment:
Added links for all of these (Brotli is well known, but I hadn't heard of UCL or LZO)